src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
```

Comment:
The import statements should be in the following order:
import static ...
<blank line>
import ...